dbt_project.yml

```diff
@@ -823,12 +823,6 @@ models:
 
     contracts:
       +schema: contracts
```

Comment:
dbt compile is throwing warnings on this still being here, as no objects exist there anymore:
https://github.com/duneanalytics/spellbook/actions/runs/6935445414/job/18865590337?pr=4747#step:8:12

you can remove these two lines too, it seems.

Reply:
removed